frontend/src/components/editable-cell.tsx

```diff
@@ -81,6 +83,12 @@ export function EditableCell<T extends HasId, Field extends keyof T = keyof T>({
     }
 
     const formattedValue = isDate ? ("" + value).slice(0, 10) : value || "";
```

Comment:
What's the difference between `formattedValue` and `displayValue`? If they're both needed, then there should be some comments explaining what they're doing.